apps/api/src/server/applications/application/application.service.js

```diff
@@ -127,26 +127,32 @@ const defaultInclusions = {
 	gridReference: true
 };
 
+/**
+ *
+ * @param {{subSector?: any, sector?: any, caseEmail?: any, keyDates?: any, geographicalInformation?: any, locationDescription?: any, regions?: any, status?: any, applicants?: any, applicantsAddress?: any}} query
+ * @returns {object}
+ */
 const inclusionsUsingQuery = (query) => {
 	return {
 		subSector: notFalseOrUndefined(query?.subSector) || notFalseOrUndefined(query?.sector),
 		sector: notFalseOrUndefined(query?.sector),
 		applicationDetails:
 			notFalseOrUndefined(query?.keyDates) ||
-			notFalseOrUndefined(query?.geographicalInformation?.locationDescription),
+			notFalseOrUndefined(query?.geographicalInformation) ||
+			notFalseOrUndefined(query?.caseEmail),
 		zoomLevel: notFalseOrUndefined(query?.geographicalInformation),
 		regions:
 			notFalseOrUndefined(query?.regions) || notFalseOrUndefined(query?.geographicalInformation),
 		caseStatus: query?.status,
-		serviceCustomer: notFalseOrUndefined(query.applicant),
-		serviceCustomerAddress: notFalseOrUndefined(query?.applicant?.address),
+		serviceCustomer: notFalseOrUndefined(query?.applicants),
+		serviceCustomerAddress: notFalseOrUndefined(query?.applicantsAddress),
 		gridReference: notFalseOrUndefined(query.geographicalInformation)
 	};
 };
 
 /**
  *
- * @param {{subSector?: any, sector?: any, keyDates?: any, geographicalInformation?: any, regions?: any, status?: any, applicant?: any}} query
+ * @param {{subSector?: any, sector?: any, caseEmail?: any, keyDates?: any, geographicalInformation?: any, regions?: any, status?: any, applicants?: any, applicantsAddress?: any}} query
```

Comment:
Same as for `inclusionUsingQuery` we can probably replace all the `any`s with `boolean | object`